dlk/python/dlk/templates/include/global.tpl.h

```diff
@@ -30,12 +30,32 @@ typedef int16_t   T_INT16;
 
 #define QUANTIZED_NOT_PACKED uint8_t
 
+enum class MemoryFormat {
+  HWC,
+};
+
+template <typename pack_type, MemoryFormat memory_format>
+class QuantizedPacked {
```

Comment:
Also would put `__attribute__ ((packed))` here to be sure that `QuantizedPacked` is always the same thing as `pack_type` in terms of in-memory representation.

Reply:
Add attribute `packed`.